data-visualize.qmd

```diff
@@ -124,65 +122,59 @@ ggplot(penguins, aes(x = species)) +
   geom_bar(fill = "red")
 ```
 
-3.What does the `bins` argument in [`geom_histogram()`](https://ggplot2.tidyverse.org/reference/geom_histogram.html) do?
-
-It determines the width or size of the bars.
+3. It determines the number of bins.
 
-4.Make a histogram of the `carat` variable in the `diamonds` dataset. Experiment with different binwidths. What binwidth reveals the most interesting patterns?
+4.
 
 ```{r}
-ggplot(diamonds,aes(x = carat)) + geom_histogram(binwidth = .10)
+ggplot(diamonds,aes(x = carat)) +
+  geom_histogram(binwidth = 0.10)
 ```
 
 I think .10 binwidth shows the most results.
 
 ## 2.5.5 Exercises {.unnumbered}
 
-1.  Which variables in `mpg` are categorical? Which variables are continuous? (Hint: type [`?mpg`](https://ggplot2.tidyverse.org/reference/mpg.html) to read the documentation for the dataset). How can you see this information when you run `mpg`?
-
-6 categorical,displ is continous,glimpse(mpg) or type mpg or the help function ?mpg
-
-2.  Make a scatterplot of `hwy` vs. `displ` using the `mpg` data frame. Next, map a third, numerical variable to `color`, then `size`, then both `color` and `size`, then `shape`. How do these aesthetics behave differently for categorical vs. numerical variables?
+1. Manufacturer,class,fl,drv,model,and trans are all categorical variables.Displ,year,cyl,cty,and hwy are all numerical variables. You can run `glimpse` `mpg` or run ?mpg to see the variables.
+2.
```

Comment:
The question "How do these aesthetics behave differently for categorical vs. numerical variables?" isn't answered